data.js

```diff
@@ -299,38 +317,6 @@ const pages = [
       'JavaScript',
     ],
   },
-  {
```

Comment:
@wesbos similar to the addition above, idk why this deletion is here. Feel free to re-add it if necessary.